onnx2torch/node_converters/activations.py

```diff
@@ -116,6 +119,36 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
     )
 
 
+class OnnxPReLU(nn.Module, OnnxToTorchModuleWithCustomExport):  # pylint: disable=missing-docstring
```

Comment:
Thank you for pull request.
Please add test for this activation function in activation tests: https://github.com/ENOT-AutoDL/onnx2torch/blob/develop/tests/node_converters/activations_test.py .

Reply:
Hi @irkjero , I've added the tests for PRelu.